sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -382,6 +382,8 @@ public:
   node add(T cgf, const property_list& propList = {});
 
   void make_edge(node& src, node& dest);
+
+  void print_dot_graph(std::string path, bool verbose = false) const;
```

Comment:
After looking at this again, I'm wondering if it would be better not encoding the file format in the function name. Let's say we pick a more generic function name like `print_graph` we could derive the format from the file ending, or consider other options. This would give us the opportunity easily adding support for other formats in the future.

Reply:
"dot" has been removed from the function name and the extension is now extracted from the file name.